.github/workflows/acceptance-tests-on-emulator.yaml

```diff
@@ -18,16 +18,16 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        ruby: [2.6, 2.7, 3.0]
-        ar: [6.0.5.1, 6.1.6.1, 7.0.2.4, 7.0.3.1]
+        ruby: ["2.6", "2.7", "3.0", "3.1"]
+        ar: ["~> 6.0.6", "~> 6.1.7", "~> 7.0.4"]
```

Comment:
Would it make even more sense to change this to:

```suggestion
        ar: ["~> 6.0", "~> 6.1", "~> 7.0"]
```

?

Reply:
@olavloite "~> 6.0" installes the latest 6.x.

Reply:
Ah, thanks! I misunderstood that. So what we had was already what we wanted for that. I'll make a fix for that.